### ISSUE TITLE
McAfee keeps detecting a threat and wont let me download

### ISSUE THREAD

**Issue by Assassin-of-Red07** [LostDragonist/steam-library-setup-tool#11]
![Screenshot 2024-04-20 111734](https://github.com/LostDragonist/steam-library-setup-tool/assets/167633039/30ba9565-9940-480e-8cc3-2b3749b28e41)

McAfee wont let me download

**Comment by LostDragonist** [comment]
Yeah, it'll do that. The method I'm using to package the program is often incorrectly flagged by anti-virus programs as the entire reason the program exists is to be an executable that modifies system files.

There might be a better way to write the program that doesn't cause these issues but I don't know what it is.  Or really have the time/motivation to care.

If you're having problems disabling McAfee or allowing it to download the file, I suggest you contact McAfee customer support.

**Comment by Assassin-of-Red07** [comment]
Thank you for quick reply! 

I am fairly new to modding so I am trying to do the Fallout "Begin Again" on Wabbajack/Nexus so trying to follow the instructions and don't want to mess it up. I will temp disable mcafee and try again.
